k8s/aggregator/development/config-release-aggregator.toml

```diff
@@ -0,0 +1,14 @@
+[logging]
+filter = "info,xain_fl=trace"
+telemetry = { jaeger_endpoint = "jaeger:6831", service_name = "aggregator" }
```

Comment:
We should be good to get rid of this, at least on `k8s`'s scope...

Reply:
Good catch, I've already forgot about it. Done!